_api-reference/cluster-decommission.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Cluster decommission 
+nav_order: 20
+---
+
+# Cluster decommission
+
+The cluster decommission operation adds support decommissioning based on awareness. It greatly benefits uses with multi-zone deployments, where awareness attributes, such as `zones`, can aid in applying new upgrades to a cluster in controlled fashion. This is especially useful during outages, in which case, you can decommission that unhealthy zone to prevent replication requests from getting stalled and prevent your request backlog from becoming to large.
+
+For more information on allocation awareness, see [Shard allocation awareness]({{site.url}}{{site.baseurl}}//opensearch/cluster/#shard-allocation-awareness)
+
+
+## Path and HTTP methods
```

Comment:
This is "HTTP and path methods" above. Please align with template in both instances.